shared/checks/oval/file_permissions_unauthorized_sgid.xml

```diff
@@ -48,14 +55,16 @@
     <value>/usr/bin/wall</value>
     <value>/usr/bin/write</value>
     <value>/usr/lib/vte/gnome-pty-helper</value>
+    {{% if product != "rhel6" %}}
```

Comment:
I think it would be better to move these values inside `else` on line 42, and remove this `if`. That would reduce number of conditions and improve readability.